inst/include/distributions/functors/lognormal_lpdf.hpp

```diff
@@ -50,17 +46,24 @@ namespace fims_distributions
          */
         virtual const Type evaluate()
         {
-            this->logmu.resize(this->x.size());
-            this->logsd.resize(this->x.size());
-            is_na.resize(this->x.size());
-            this->lpdf_vec.resize(this->x.size());
-            for (size_t i = 0; i < this->x.size(); i++)
+          size_t n_x;
```

Comment:
🥳 🎉 Thanks Andrea for refactoring the distribution codebase! I’m excited to try out the new interface.